server/src/main/java/org/elasticsearch/search/aggregations/support/MultiValuesSource.java

```diff
@@ -55,6 +57,46 @@ public SortedNumericDoubleValues getField(String fieldName, LeafReaderContext ct
         }
     }
 
+    public static class AnyMultiValuesSource extends MultiValuesSource<ValuesSource> {
+        public AnyMultiValuesSource(Map<String, ValuesSourceConfig> valuesSourceConfigs, QueryShardContext context) {
```

Comment:
maybe we should just call it `GeoLineMultiValueSource`? It is only used by that aggregation which expects a numeric field and a geo point field.

Reply:
great idea. will change it!